### PR TITLE
Updates documentation for Principles and Environments

### PR DESCRIPTION
**Pull request by nikola-nulogy**
Also updates styling and the logo.

### CODE REVIEW

File: catalog/foundation/principles.md [hunk 1]
```diff
@@ -4,18 +4,13 @@
...
-## Decisive
-* Empower user decisions with data
-* Just enough information; always helpful
+## Just enough information; always helpful
+Enterprise systems are burdened with overloaded pages. We seek to find the “contextual balance” where information is helpful providing decision support, recommendations and insights.
 
-## Reassuring
-* Connect user and system
-* Dependable stability
+## Adaptive Experience
```
Comment (theinterned) [inline catalog/foundation/principles.md]: Is the description missing here?

File: catalog/index.js [hunk 2]
```diff
@@ -9,19 +9,23 @@ const config = {
...
   theme: {
-    fontFamily: "'Rubik', serif",
-    fontHeading: "'Rubik', serif",
-    background: '#F3F1F2',
-    textColor: '#58595b',
-    codeColor: '#1C68A5',
-    linkColor: '#1C68A5',
-    linkHoverColor: '#0E77D2',
+    //fontFamily: "'Rubik', serif",
+    //fontHeading: "'Rubik', serif",
```
Comment (theinterned) [inline catalog/index.js]: I noticed these lines were commented out and restored them.

File: catalog/foundation/environments.md [hunk 3]
```diff
@@ -1,11 +1 @@
...
->Think about direction
->Wonder why you haven't before
->
-> Now stand in the place where you work
-> Now face west
-> Think about the place where you live
-> Wonder why you haven't before
->
-> — [R.E.M. "Stand"](https://www.youtube.com/watch?v=AKKqLl_ZEEY)
+<div><img src="/environments.png" /></div>
```
Comment (theinterned) [inline catalog/foundation/environments.md]: do we want this image to be responsive in any way? at least with a `width: 100%;`?

Comment (theinterned) [reply catalog/foundation/environments.md]:
```css
width: 100%;
min-width: 500px;  // or whatever
```

Also, what is the div for?

Comment (theinterned) [reply catalog/foundation/environments.md]: @nikola-nulogy did you consider using the [Image speciman](https://docs.catalog.style/specimens/image)?

**Review by theinterned — CHANGES_REQUESTED** [review]
@nikola-nulogy looks good: I left a few comments that it would be good to address before merging

File: catalog/foundation/environments.md (hunk 3 above)
Comment (nikola-nulogy) [reply catalog/foundation/environments.md]: I didn't and probably should have. I guess this was me just playing around. I will have a look at image specimen and provide a proper implementation.

File: catalog/foundation/principles.md (hunk 1 above)
Comment (nikola-nulogy) [reply catalog/foundation/principles.md]: We do not have properly written description for this one that's why it is not included.

**Review by theinterned — APPROVED** [review]

**Pull request merged**